website/versioned_docs/version-2.0.0/install/install-with-script.md

```diff
@@ -128,7 +123,263 @@ mkdir prysm && cd prysm
+
+```text
+./prysm.sh beacon-chain --http-web3provider=<YOUR_ETH1_NODE_ENDPOINT>
+```
+
+**Prater**
+
+Download the genesis state from [github.com/eth2-clients/eth2-networks/blob/master/shared/prater/genesis.ssz](https://github.com/eth2-clients/eth2-networks/blob/master/shared/prater/genesis.ssz) to a local file, then run
+
+```text
+./prysm.sh beacon-chain --http-web3provider=<YOUR_ETH1_NODE_ENDPOINT> --prater --genesis-state=/path/to/genesis.ssz
+```
+
+</TabItem>
+</Tabs>
+
+## Running a Validator
+
+A validator is an optional process that can be attached to a running beacon node to stake your ETH and participate in the chain's consensus. It is the analogue of a **miner** from proof-of-work-based systems.
+
+### Step 1: Ensure your beacon node is synced
+
+An important step in the process is ensuring your beacon node is all set up before trying to run a validator. This is because after your validator is inducted into the participating validator set, it is expected to begin performing its duties almost right away. It is important to run a validator with a node that is synchronized to the chain head so you can start earning ETH instead of losing it.
+
+:::tip Syncing your node
+The beacon-chain node you are using should be **completely synced** before submitting your deposit. You may **incur minor inactivity balance penalties** if the validator is unable to perform its duties by the time the deposit is processed and activated by the beacon chain network.
+:::
+
+You can check the sync status of your node with the following command on most systems:
+
+```text
+curl http://localhost:3500/eth/v1alpha1/node/syncing
+```
+
+If your node is done synchronizing, you will see the response:
+
+```text
+{"syncing":false}%
+```
+
+### Step 2: Send your validator deposit via the Ethereum validator launchpad
+
+:::danger Ensure You Are Not Being Scammed
+The correct address for the launchpad is https://launchpad.ethereum.org and the only, official validator deposit contract is [0x00000000219ab540356cbb839cbe05303d7705fa](https://etherscan.io/address/0x00000000219ab540356cbb839cbe05303d7705fa). Do not send ETH directly to the contract, and only join by using the eth2 launchpad.
+:::
+
+The [Official Eth2 Launchpad](https://launchpad.ethereum.org/summary) is the easiest way to go through a step-by-step process to deposit your 32 ETH to become a validator. Throughout the process, you'll be asked to generate new validator credentials using the official Ethereum deposit command-line-tool [here](https://github.com/ethereum/eth2.0-deposit-cli). Make sure you use the `mainnet` option when generating keys with the deposit CLI. During the process, you will have generated a `validator_keys` folder under the `eth2.0-deposit-cli` directory. You can import all of your validator keys into Prysm from that folder in the next step.
```

Comment:
again for testnet either the updated url or the mention on switching the inside the website ( top right corner) for prater